internal/versions/versions_test.go

```diff
@@ -19,39 +20,125 @@ func TestHandler_ListComponentVersions(t *testing.T) {
 
 var _ = Describe("list versions", func() {
```

Comment:
Just an idea, do you think that it worth testing api level code with `httptest`?

Reply:
I think that only when we want to specifically test server behavior we need that, no?
Otherwise, if it directly call the Golang functions, it's the same UT